docs/open-source-tizen/tizen-rt/release-notes/tizen-rt-1-0-m1.md

```diff
@@ -1,18 +1,16 @@
-# Tizen RT 1.0 M1
+# Tizen RT 1.0 M1 Release Notes
 
 Release Date: 23 Dec, 2016
 
-## Release Notes
-
-**New and Changed Features**
+## New and Changed Features
 
 - Architecture
   - ARM
     - Cortex-R4
       - SIDK-S5JT200
 - Kernel
   - FIFO / RoundRobin Scheduling and Interrupt
-    - Fully Preemptible
+    - Fully-preemptible
```

Comment:
First of all, editing is an on-going task. No matter how many times you edit something, when you look at it again after some time has passed, you will notice new issues that could be improved.
So no document is ever perfect after a single editing round. :-)

Since the release notes are visible on the site, we think that it is important to keep them up-to-date style-wise, and when we read them through and notice some room for improvement, we edit them. Of course we do not modify the factual content of a release note after the release has been published. But improving the language and making the release note more readable should be acceptable to you.

Are you now saying that we are not allowed to touch release notes, except to fix a glaring typo?
If you are, does this also apply to general style changes that happen in the library all the time? 
For example, when trademarks were missing from certain products and we added and fixed those all over, we also fixed them in the release notes. In the future, would those kind of fixes be forbidden for release notes?

Before we go back to the original version of these release notes, please clarify what exactly we are allowed to do with release notes in the future. 
And if you say that we are only allowed to touch typos, what exactly do you mean by a typo?
For example, the fix above ("Fully Preemptible" > "Fully-preemptible") is a typo for us, since it fixes the way the word has been written. If this is not a typo for you, then we need some further clarifications on what exactly is a typo. Do you mean a situation where a wrong word is used (like when "out" is written "our", causing the meaning of the sentence to break down)?

Or would it be easiest to just agree that we will never touch a published release note for any reason? At least then we will not make any changes that you feel are undesirable.